book/src/pong-tutorial/pong-tutorial-04.md

```diff
@@ -0,0 +1,351 @@
+# Making a ball move and bounce
+
+In the previous chapter, we learned how to capture user input
+to make things move on the screen by creating a so-called System
+ourselves.
+This chapter will reuse all the knowledge we acquired through the
+previous chapters to add a new object to our game: a ball that moves
+and bounces around!
+
+First, let's define some other useful constants for this chapter:
+```rust,no_run,noplaypen
+pub const BALL_VELOCITY_X: f32 = 75.0;
+pub const BALL_VELOCITY_Y: f32 = 50.0;
+pub const BALL_RADIUS: f32 = 2.0;
+```
+
```

Comment:
Maybe point out this could be placed in a config file for when you want to edit these values a lot? Or probably the pong example should be changed to using that?

Reply:
I don't know if it is sensible to add this comment here as it is the way it's done in previous chapters. But I may update those as well? Also, what about making a chapter for this kind of "refactors" later?

Reply:
:+1: I think a simple pointer here would be enough.

Reply:
something like "This could also be done by .., but we're keeping it simple here."